src/components/theme/Navigation/ContextNavigation.stories.js

```diff
@@ -0,0 +1,216 @@
+    {
+      '@id': 'http://localhost:3000/api/folder1',
+      description: '',
+      href: 'http://localhost:3000/api/folder1',
+      icon: '',
+      is_current: false,
+      is_folderish: true,
+      is_in_path: false,
+      items: [],
+      normalized_id: 'folder1',
+      review_state: 'private',
+      thumb: '',
+      title: 'Folder1',
+      type: 'folder',
+    },
+    {
+      '@id': 'http://localhost:3000/api/folder2/folder21',
+      description: '',
+      href: 'http://localhost:3000/api/folder2/folder21',
+      icon: '',
+      is_current: false,
+      is_folderish: true,
+      is_in_path: true,
+      items: [
+        {
+          '@id': 'http://localhost:3000/api/folder2/folder21/doc211',
+          description: '',
+          href: 'http://localhost:3000/api/folder2/folder21/doc211',
+          icon: '',
+          is_current: false,
+          is_folderish: false,
+          is_in_path: false,
+          items: [],
+          normalized_id: 'doc211',
+          review_state: 'private',
+          thumb: '',
+          title: 'Doc211',
+          type: 'document',
+        },
+        {
+          '@id': 'http://localhost:3000/api/folder2/folder21/doc211-copy',
+          description: '',
+          href: 'http://localhost:3000/api/folder2/folder21/doc211-copy',
+          icon: '',
+          is_current: true,
+          is_folderish: false,
+          is_in_path: false,
+          items: [],
+          normalized_id: 'doc212',
+          review_state: 'private',
+          thumb: '',
+          title: 'Doc212',
+          type: 'document',
+        },
+      ],
+      normalized_id: 'folder21',
+      review_state: 'private',
+      thumb: '',
+      title: 'Folder21',
+      type: 'folder',
+    },
+  ],
+  title: 'NavTree',
+  url: 'http://localhost:3000/api/folder2/sitemap',
+};
+
+const customStore = {
+  contextNavigation: {
+    '/folder2/folder21/doc212/@contextnavigation': {
+      data: navigation,
+    },
+  },
+  userSession: { token: '1234' },
+  intl: {
+    locale: 'en',
+    messages: {},
+  },
+};
+
+const ContextNavigationComponent = (args) => {
+  return (
+    <Wrapper>
+      <style dangerouslySetInnerHTML={{ __html: style }}></style>
+      <CNC {...args} navigation={navigation} />
+    </Wrapper>
+  );
+};
+
+const ContextNavigation = (args) => {
+  return (
+    <Wrapper
+      location={{ pathname: '/folder2/folder21/doc212' }}
+      customStore={customStore}
+    >
+      <style dangerouslySetInnerHTML={{ __html: style }}></style>
+      <ContextNavigationDefault />
+    </Wrapper>
+  );
+};
+
+export default {
+  title: 'Context navigation',
+  component: CNC,
+  decorators: [
+    (Story) => (
+      <div style={{ margin: '3em' }}>
+        <Story />
+      </div>
+    ),
+  ],
+  // subcomponents: { ArgsTable },
+};
+
+export const Renderer = () => (
+  <ContextNavigationComponent navigation={navigation} />
+);
+
+export const Connected = () => <ContextNavigation />;
+
+const style = `
```

Comment:
Why the style is needed?

Reply:
There's no style in Volto right now for the NavPortlet.